activesupport/lib/active_support/dependencies.rb

```diff
@@ -760,7 +736,6 @@ def remove_constant(const) #:nodoc:
         begin
           constantized = parent.const_get(to_remove, false)
         rescue NameError
-          log "the constant #{const} is not reachable anymore, skipping"
```

Comment:
@aaronang I think it's this one, the matching one below, and probably the "removing partially loaded constants" thing further up

:heart: